.github/workflows/compute.yml

```diff
@@ -30,6 +30,30 @@ jobs:
       NFT_CONTRACT_ADDRESS: ${{ secrets.NFT_CONTRACT_ADDRESS }}
 
     steps:
+      - name: Post starting comment to issue
+        uses: actions/github-script@v7
+        id: post-comment
+        with:
+          github-token: ${{ inputs.authToken }}
+          script: |
+            const comment_body = '\`\`\`diff\n+ Evaluating results. Please wait...';
+            const obj = ${{ inputs.eventPayload }}
+            if (obj.issue && "${{ inputs.eventName }}" === "issues.closed") {
```

Comment:
It seems to make more sense to write all the logic inside of typescript scripts. You can use the tsx runtime. 

This is the reason why I thought this might be a v1 bot change. It doesn't make sense to me to mix GitHub actions logic as part of the bot given that we don't seem to do this anywhere else that I've seen. 

https://github.com/ubiquibot/conversation-rewards/issues/29#issuecomment-2219609081

Reply:
This is what I did at first but running a script requires:
- cloning the repo
- installing node
- install the packages

By the time this is done, there is around 2 seconds left for the whole action to finish so the waiting message is pointless.

The point of scripting it right inside the action is to avoid all the previous steps and gain around 15 seconds of execution.

If you really don't want this approach, what could be done os:
- totally remove the waiting message
- create our own action step that packages our logger

Reply:
Let's try this approach then. I understand now.